### ISSUE TITLE
Re-enable BTCpay ?

### ISSUE THREAD

**Issue by Jpja** [CounterpartyXCP/counterwallet#830]
In the DEX module it is possible to place order with token `BTC`.

However, when I did place a matching order and it confirmed, I was the one supposed to transfer BTC. CW did not do this although I stayed logged in, and even tried to log in and out, etc.

We should consider the following options:

1) Disable BTC from the token selection menu
2) Re-enable BTCpay (it was disabled, possibly just a parameter to re-enable it)
3) Re-enable BTCpay with some UI/UX upgrades

Since there is popular demand for BTC on DEX and FreeWallet will soon launch with BTCpay, I suggest we look into the 3rd option.

Perhaps, when the user is the one supposed to transfer BTC, only allow him to place order if it appears to match existing orders.

**Comment by informatorius** [comment]
Is this easy to fix? Some users still try to match orders with BTC and don't know about BTCpay not working - so these fail.

**Comment by chiguireitor** [comment]
Hmmm.... i would most likely remove the possibility to open BTCPays for now from CW, there needs to be a HUGE disclaimer about btcpays and how they work.

**Comment by informatorius** [comment]
Freewallet.io already has implemented automatic BTCpay in paid version.

**Comment by chiguireitor** [comment]
Indeed.... it's kinda cumbersome to implement and there's need for someone to actively maintain CW (and i'm bogged down with lib/cli + fednode + indexd work)

**Comment by informatorius** [comment]
Then option 1. Disable BTC from token selection is maybe the next step to go.

**Comment by chiguireitor** [comment]
Yes, temporarily at least.

Less user friction is the way to go while we get enough developers to handle all the code.

**Comment by jdogresorg** [comment]
As @chiguireitor has mentioned, the issue is not with Counterparty but rather with counterwallet. The main issue being that when BTCpay is enabled in counterwallet, the user has to keep their browser window open to sign BTCpay transactions, which is less than ideal.

Freewallet.io already has support for auto and manual BTCpay transactions in the Desktop version and a fork of @wasthatawolf btcpaymarket.com codebase has been forked to https://btc2xcp.com/ and handles trading with BTC on the CP DEX.

I also think that disabling BTC from the token selection in counterwallet is the simplest fix in the short-term.
